tooldb.py

```diff
@@ -0,0 +1,413 @@
+"""
+<Program Name>
+  tooldb.py
+
+<Author>
+  Lukas Puehringer <lukas.puehringer@nyu.edu>
+
+<Started>
+  June 12, 2017
+
+<Copyright>
+  See LICENSE for licensing information.
+
+<Purpose>
+  A basic collection of software supply chain tools in four categories
+   - vcs (version control systems)
+   - building
+   - qa (quality assurance)
+   - package (packaging)
+
+  The tools are presented to the user on the different pages of the web wizard
+  as options to choose from to define a custom supply chain
+
+  TODO:
+  - Update! Some of the tools might be not used at all while other popular
+    tools are missing
+  - Clean up! Common commands, logo, ...
+
+"""
```

Comment:
I'm not exactly fond of having a .py file with a constant value in lowercase and a bunch of commented out stuff. I suggest we either move to a just-data representation format and/or make the variable be all uppercase.

Reply:
Making the constant name uppercase makes sense. Also, the commented out stuff will disappear (cf. https://github.com/in-toto/layout-web-tool/issues/2).
What's your concern about the file being a Python file?

Reply:
Well, It's just a script that's imported to define a constant. I was imagining this, in the future, could be a dataset that's fed to the webapp, rather than a script that a developer/maintainer has to update.

I can totally see this being maintained as a csv in the future....

Reply:
Tempfixed with ebf524dd1293ee5b361c41870239f0e0e019bfa2